forge-game/src/main/java/forge/game/GameAction.java

```diff
@@ -583,10 +583,15 @@ private Card changeZone(final Zone zoneFrom, Zone zoneTo, final Card c, Integer
             }
         }
 
-        // Need to apply any static effects to produce correct triggers
-        checkStaticAbilities();
+        if (!table.isEmpty()) {
+            // we don't want always trigger before counters are placed
+            game.getTriggerHandler().suppressMode(TriggerType.Always);
```

Comment:
When you cast _Endless One_ with X=1 while your opponent controls _Vorinclex, Monstrous Raider_:
the trigger would stay supressed

Reply:
fixed